src/jsdoc_transformer.ts

```diff
@@ -722,8 +722,7 @@ export function jsdocTransformer(
         // forces any global declarations from the library to be visible, which is what users use
         // this for. No symbols from the script need forward declaration, so just return.
         if (!sym) return importDecl;
-        // Write the export declaration here so that forward declares come after it, and
```

Comment:
FYI this comment was just a misleading leftover, unrelated to the main change here.